### PR TITLE
Fix #1587: Made intent/bundle/saved/tags instance keys consistent.

### LINKED ISSUE #1587: Make intent/bundle/saved/tags instance keys consistent.
The keys should be consistent in intents, bundles, tags, etc. They should mainly follow below mentioned structure.

For fragments:
`<PROPERTY_NAME>_ARGUMENT_KEY = "<FileName>.<property_name>"`
Example: `INTERNAL_PROFILE_ID_ARGUMENT_KEY = "SampleFragment.internal_profile_id"`

For activities:
`<PROPERTY_NAME>_EXTRA_KEY = "<FileName>.<property_name>"`
Example: `INTERNAL_PROFILE_ID_EXTRA_KEY = "SampleActivity.internal_profile_id"`

For saved instances:
`<PROPERTY_NAME>_SAVED_KEY = "<FileName>.<property_name>"`
Example: `INTERNAL_PROFILE_ID_SAVED_KEY = "SampleActivity.internal_profile_id"`

### PR DESCRIPTION
**Pull request by ayush0402**
## Explanation

Fixes #1587
Made KEYS consistent in : 

- CompletedStoryListActivity.kt
- DeveloperOptionsTestActivity.kt
- MarkChaptersCompletedActivity.kt
- MarkChaptersCompletedFragment.kt
- MarkChaptersCompletedTestActivity.kt
- MarkStoriesCompletedActivity.kt
- MarkStoriesCompletedFragment.kt
- MarkStoriesCompletedTestActivity.kt
- MarkTopicsCompletedActivity.kt
- MarkTopicsCompletedFragment.kt
- MarkTopicsCompletedTestActivity.kt
- ProfileProgressActivity.kt
- ProfileProgressFragment.kt

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

### CODE REVIEW

**Comment by ayush0402** [comment]
@rt4914 PTAL. I tried renaming the everything left trying to conclude this issue.

File: app/src/main/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivity.kt [hunk 1]
```diff
@@ -16,19 +16,19 @@ class CompletedStoryListActivity : InjectableAppCompatActivity() {
     super.onCreate(savedInstanceState)
     (activityComponent as ActivityComponentImpl).inject(this)
     val internalProfileId: Int =
-      intent.getIntExtra(COMPLETED_STORY_LIST_ACTIVITY_PROFILE_ID_KEY, -1)
+      intent.getIntExtra(COMPLETED_STORY_LIST_ACTIVITY_PROFILE_ID_EXTRA_KEY, -1)
```
Comment (anandwana001) [inline app/src/main/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivity.kt:19]: As per mentioned in the issue description:

```
For activities:
<PROPERTY_NAME>_EXTRA_KEY = "<FileName>.<property_name>"
Example: INTERNAL_PROFILE_ID_EXTRA_KEY = "SampleActivity.internal_profile_id"
```

we are including only `<PROPERTY_NAME>_EXTRA_KEY` not the activity name - `COMPLETED_STORY_LIST_ACTIVITY`!!

Deferring to @rt4914 for clarification.

**Review by anandwana001 — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivity.kt (hunk 1 above)
Comment (rt4914) [reply app/src/main/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivity.kt:19]: Make sense. It should be 

`const val PROFILE_ID_EXTRA_KEY = "CompletedStoryListActivity.profile_id"

Comment (ayush0402) [reply app/src/main/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivity.kt:19]: @anandwana001 @rt4914 PTAL here #3880

**Pull request closed without merging**